policies/coco-transition-checklist.md

```diff
@@ -5,7 +5,8 @@ as needed.
 ### For incoming CoCo member(s):
 
 - [ ] Add to Google Shared Drive
-- [ ] Create new Google Doc running notes
+- [ ] Create new Google Doc running notes and update link on policies README
+    - Example: [astropy-project#463](https://github.com/astropy/astropy-project/pull/463)
```

Comment:
```suggestion
- [ ] Create new Google Doc running notes and update links on policies README and Roles page
    - Example: [astropy-project#463](https://github.com/astropy/astropy-project/pull/463) and [astropy.github.com#646](https://github.com/astropy/astropy.github.com/pull/646)
```
may want to consolidate this with the entry `Update Roles page` below somehow.

Reply:
Why do we have to move the Role stuff here? This entry is about the running notes.

Reply:
Yes, and `roles.json` has a link to those that needs to be updated as well.